build/scripts/build.ps1

```diff
@@ -488,7 +488,13 @@ function Test-XUnit() {
         }
     }
     elseif ($testVsi) {
-        $dlls = Get-ChildItem -re -in "*.IntegrationTests.dll" $unitDir
+        # Since they require Visual Studio to be installed, ensure that the MSBuildWorkspace tests run along with our VS
+        # integration tests in CI.
+        if ($cibuild) {
+            $dlls += @(Get-Item (Join-Path $unitDir "Workspaces.MSBuild.Test\Microsoft.CodeAnalysis.Workspaces.MSBuild.UnitTests.dll"))
```

Comment:
I'm pretty sure it'll stop running the tests. Is that right @jaredpar?

Reply:
Wohoo. Five minutes after loading the PR and I can finally comment here. 

Oh crap I pressed Enter, another five second delay. 

Anyways yes this will fail if the file is missing.

Reply:
Oh good. That's what we want then.